app/shared/src/commonMain/kotlin/ui/subject/episode/video/settings/DanmakuRegexFilterGroup.kt

```diff
@@ -53,60 +56,48 @@ internal fun isValidRegex(pattern: String): Boolean {
 
 @Composable
 internal fun SettingsScope.DanmakuRegexFilterGroup(
-    danmakuRegexFilterList: List<DanmakuRegexFilter>,
-    onAdd: (filter: DanmakuRegexFilter) -> Unit,
-    onEdit: (id: String, new: DanmakuRegexFilter) -> Unit,
-    onRemove: (filter: DanmakuRegexFilter) -> Unit,
-    onSwitch: (fiter: DanmakuRegexFilter) -> Unit,
+    danmakuRegexFilterState: DanmakuRegexFilterState,
     isLoadingState: Boolean
 ) {
     var showAdd by rememberSaveable { mutableStateOf(false) }
     var errorMessage by rememberSaveable { mutableStateOf("") }
+    var isError by rememberSaveable { mutableStateOf(false) }
 
     if (showAdd) {
         AddRegexFilterDialog(
-            onDismissRequest = { showAdd = false },
+            onDismissRequest = {
+                showAdd = false
+                isError = false
+            },
             onConfirm = { name, regex ->
                 if (regex.isBlank()) {
-                    showAdd = false
+                    errorMessage = "正则不能为空"
+                    isError = true
                 } else {
                     if (!isValidRegex(regex)) {
                         errorMessage = "正则输入法不正确"
+                        isError = true
                     } else {
-                        onAdd(
+                        danmakuRegexFilterState.addDanmakuRegexFilter(
                             DanmakuRegexFilter(
                                 id = Uuid.randomString(),
                                 name = name,
                                 regex = regex,
                             ),
                         )
                         showAdd = false
+                        isError = false
                     }
                 }
             },
             title = { Text("添加正则过滤器") },
-            description = {
-                Text("请正确添加正则表达式，例如：第一个字符为数字：'^[1-9]{1}\$'.\n匹配该正常表达式的弹幕将不会显示")
-            },
-
-            )
-    }
-
-    if (errorMessage.isNotBlank()) {
-        AlertDialog(
-            onDismissRequest = { errorMessage = "" },
-            confirmButton = {
-                TextButton(onClick = { errorMessage = "" }) {
-                    Text("确认")
-                }
-            },
-            title = { Text("错误") },
-            text = { Text(errorMessage) },
+            isError = isError,
+            errorMessage = errorMessage,
         )
     }
 
     Group(
-        title = { Text("正则过滤器管理") },
+        title = { Text("正则弹幕过滤器管理", color = MaterialTheme.colorScheme.onSurface) },
```

Comment:
```suggestion
        title = { Text("弹幕正则过滤器管理", color = MaterialTheme.colorScheme.onSurface) },
```

这样会好点吧

因为之后可能要加 "弹幕云过滤设置" 之类的